tests/bluetooth/tester/src/btp_csis.c

```diff
@@ -61,6 +63,18 @@ static uint8_t csis_get_member_rsi(const void *cmd, uint16_t cmd_len,
 	return BTP_STATUS_VAL(err);
 }
 
+static uint8_t csis_set_sirk_type(const void *cmd, uint16_t cmd_len, void *rsp,
+				      uint16_t *rsp_len)
+{
+	const struct btp_csis_set_sirk_type_cmd *cp = cmd;
+
+	LOG_DBG("Change SIRK type to encrypted");
+
+	enc_sirk = cp->type;
```

Comment:
This may be a bit too generic. 

`type` is defined as a uint8_t, but clearly used as a bool here (so type == 0x00 == not encrypted). 

If we want to keep it named `type`, then please define the possible types. 
Alternatively we can give it a boolean name like `uint8_t encrypted` so that it's more "OK" to use it as a boolean.

Reply:
Changed variable name to encrypted